src/components/AgmExtension/ExtensionRequest.vue

```diff
@@ -238,23 +239,17 @@ export default class ExtensionRequest extends Vue {
       return DateUtilities.addMonthsToDate(19, DateUtilities.dateToYyyyMmDd(this.data.incorporationDate))
     } else {
       // For subsequent AGMs, min shouldn't be later than reference date + 15 months + 1 month
-      return 'placeholder for future work'
+      return DateUtilities.addMonthsToDate(16, this.data.prevAgmDate)
     }
   }
 
-  /**
-   * Whether to disable the editing of AGM Year field.
-   * Editable (false) when Is this first AGM is set to No.
-   * Non-Editable (true) when we first load (null) or Is first AGM is Yes.
-   * Set the agmYear as incorporation date if not null and true.
```

Comment:
Please "port" these comments to the code: explain why this getter is different values depending on state.

Reply:
Yeah I was attempting to refactor that part a bit but it wasn't working. I have reverted them back to how they were for now, but I may try to get my refactored version working again later.

Reply:
That works for now.

Sometimes you have to make a real mess before you figure out how to clean it up 😆